documentation/logs.md

```diff
@@ -0,0 +1,152 @@
+# Logs
+
+Bolt supports multiple log levels. You can configure the log level from the CLI,
+or in a project configuration file. Supported logging levels, in order from most
+to least information logged, are `trace`, `debug`, `info`, `warn`, and `error`.
```

Comment:
Technically speaking `fatal` is also a valid log level, but it's only useful if you only want to see `emerg` and `critical` messages from a Puppet manifest or apply block. We don't actually use it in Bolt code. I'm not sure if that means we should include it or not - I think it's fine to leave it out.

Reply:
Yeah, Tom mentioned that we don't use it, so I opted to leave it out. The fact that it will return stuff from a puppet manifest makes me wonder if we should include it though.... Maybe I'll add a note with a link to the puppet docs for that... I'll do some digging and figure that out.

Reply:
Okay, I've added in a section for fatal logs